src/System/Taffybar/Widget/Windows.hs

```diff
@@ -73,19 +83,37 @@ windowsNew :: WindowsConfig -> TaffyIO Gtk.Widget
 windowsNew config = do
   label <- lift $ Gtk.labelNew Nothing
 
+  icon <- lift $ Gtk.imageNew
+
+  context <- ask
```

Comment:
Lets get rid of the extra newlines here. looks kinda weird as is.

Reply:
I moved the code into an auxiliary function `buildWindowsIcon` so that it looks neater.